### PR TITLE
[HUD] [BE] Add permalink to home screen

### PR DESCRIPTION
**Pull request by ZainRizvi**
Adds a permalink button at the top of the home screen, next to the branch name.

When clicked, the button offers visual feedback that the url was copied to the clipboard. The button resets after 3 seconds.

Extended the existing CopyLink component to support this use case, and also updated the icons it uses to better match standard convention

Button:
image
When clicked:
image

### CODE REVIEW

File: torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx [hunk 1]
```diff
@@ -452,6 +466,82 @@ export default function Hud() {
...
+          navigator.clipboard.writeText(url);
+          setCopied(true);
+
+          setTimeout(() => {
+            setCopied(false);
+          }, 10 * 1000); // 10000 milliseconds = 10 seconds
+        }}
+        style={style}
+      >
+        {copied ? "📋 ✅ Copied" : "📋 Permalink"}
```
Comment (seemethere) [inline torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx:539]:
```suggestion
        {copied ? "✅ Copied" : "🔗 Permalink"}
```

Just a nit, I think making these icons more clear will make it easier to understand

Comment (huydhn) [reply torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx:539]: I just realize another convention we are using for this is in minihud page https://hud.pytorch.org/minihud (Copy Link).  Click on that and the a different check mark style shows up indicating that the link has been copied.  Two thoughts:

* We should decide on one style, and make `CopyPermanentLink` reusable
* The minuhud link is broken, i.e. https://hud.pytorch.org/minihud#980f5ac0499447cd6d39b6241ae30ae30937a3e5 (not related to this change though)

Comment (ZainRizvi) [reply torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx:539]: I like it

Comment (huydhn) [reply torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx]: Weird, now it doesn't seem to crash anymore

Comment (ZainRizvi) [reply torchci/pages/hud/[repoOwner]/[repoName]/[branch]/[[...page]].tsx]: Re: choosing a convention, let's stick with the 🔗, which is more standard (and [Google agrees](https://www.google.com/search?q=usual+emoji++used+for+permalink&sca_esv=33d761b251b7bc29&sca_upv=1&ei=tvdMZoWlBsDCp84P36a66A4&ved=0ahUKEwiF2Jn2vp-GAxVA4ckDHV-TDu0Q4dUDCBA&uact=5&oq=usual+emoji++used+for+permalink&gs_lp=Egxnd3Mtd2l6LXNlcnAiH3VzdWFsIGVtb2ppICB1c2VkIGZvciBwZXJtYWxpbmsyCBAhGKABGMMEMggQIRigARjDBEiCF1CvC1ipEnADeAGQAQCYAWegAaEEqgEDNS4xuAEDyAEA-AEBmAIJoAKxBMICChAAGLADGNYEGEfCAgoQIRigARjDBBgKmAMAiAYBkAYIkgcDOC4xoAfFGQ&sclient=gws-wiz-serp))

I've updated the icons of minihud to also be the same. I can look into fixing the bug with minihud's copy link as a separate issue.

**Review by huydhn — APPROVED** [review]

**Pull request merged**